addons/account/demo/account_demo.py

```diff
@@ -30,6 +30,7 @@ def _get_demo_data(self):
     def _get_demo_data_move(self):
         cid = self.env.company.id
         ref = self.env.ref
+        fifteen_months_ago = fields.Date.today() + relativedelta(months=-15)
```

Comment:
Normally, it shouldn't be the case but are we sure this invoice isn't used anywhere for testing stuff ?